tools/weekeventregconvert.py

```diff
@@ -155,7 +155,7 @@
     (18 << 8) | 0x10: "WEEKEVENTREG_18_10",
     (18 << 8) | 0x20: "WEEKEVENTREG_18_20",
     (18 << 8) | 0x40: "WEEKEVENTREG_18_40",
-    (18 << 8) | 0x80: "WEEKEVENTREG_18_80",
+    (18 << 8) | 0x80: "WEEKEVENTREG_HAS_POWERDERKEG_PRIVILEGES",
```

Comment:
Where is this set? I'm having trouble verifying this in code. Could just leave it unnamed otherwise.

Reply:
The opening comment of this PR and reference to ([3c218d7](https://github.com/zeldaret/mm/pull/1080/commits/3c218d759ee8473ac4eadaea388fbccdf221f0cf)) explains the week events. In short, I used MsgScript parsing to figure out the purposes of the events used in code, but at the time (haven't actually confirmed recently) the code upon which it was based hadn't been committed, so it got stripped.